tools/parity_scale_decoder/src/generate_map.rs

```diff
@@ -97,7 +97,7 @@ pub fn generate_map() -> DumpDecodedMap {
         BTreeSet<PublicKey>,
         BTreeSet<RoleId>,
         BTreeSet<SignatureOf<transaction::Payload>>,
-        BlockRejectionReason,
+        block::error::BlockRejectionReason,
```

Comment:
I'm not a huge fan of how this function works. I either think that we should use fully qualified names only, or that instead we should add to this list with various scoped `use` statements.

Reply:
if any is relevant type is missing they are reported because they are taken from schemas. We probably shouldn't even be listing them manually but rather generate from schemas. In any case this is not a problem except being tedious to list them all manually